app/controllers/api/v1/search_controller.rb

```diff
@@ -619,6 +632,29 @@ def self.match_studies_by_facet(query_results, search_facets)
         matches
       end
 
+      # Simplify TDR results to be mappable for the UI badges for facet search
+      def self.simplify_TDR_facet_search_results(query_results, search_facets)
+        simple_TDR_result = {}
+        simple_TDR_results = []
+        query_results.each do |result|
+          unless result[0].nil?
```

Comment:
I definitely would like to grab a few minutes of your time to look into this more - this shouldn't ever be the case.  If we're getting back results without accessions/short names, that's a problem.